### DISCUSSION TITLE
7.23 breaks mocha cli --require

### DISCUSSION THREAD

**Discussion by bubonicfred** [babel/babel#16002] [Q&A]
I have unit tests that are run via mocha
` npx mocha "./tests/unit/**/*.test.js" --require @babel/register --require tests/unit/babel && npm run test:fixtures:seed-minutes:unit`
Running this under 7.23 gives me the error 
`Error [ERR_REQUIRE_ESM]: require() of ES Module /home/runner/work/5minitz/5minitz/node_modules/get-func-name/index.js from /home/runner/work/5minitz/5minitz/node_modules/chai/lib/chai/utils/inspect.js not supported.`

Is there a workaround for this? 
EDIT: For whatever reason I no longer have this issue. I'll update if it decides to reappear

**Comment by nicolo-ribaudo** [comment]
Do you have a repo that reproduces the issue?

**Reply by bubonicfred** [reply]
I made a test branch to show and it appears to have sorted itself?
